portfolio/src/main/webapp/quiz.js

```diff
@@ -2,40 +2,91 @@
  * Question data type.
  */
 class Question {
-  constructor(question, answers, correct) {
+  constructor(question, answers, correctAnswerIndex) {
     this.question_ = question;
     this.answers_ = answers;
     this.correctAnswerIndex_ = correctAnswerIndex;
   }
 
   getQuestion() {
```

Comment:
From Style Guide: In methods and named functions, parameter and return types should be documented, except in the case of same-signature @overrides, where all types are omitted.

Same comment for the other methods.

Reply:
I didn't notice that while consulting the style guide, I will add the documentation.